client/src/components/TimeSlotSelector.test.jsx

```diff
@@ -2,9 +2,141 @@ import React from 'react';
 import { shallow } from 'enzyme';
 
 import TimeSlotSelector from './TimeSlotSelector';
+// import TimeSlot from './TimeSlot';
 
 describe('TimeSlotSelector Component', () => {
-  test('should render correctly', () => {
-    expect(shallow(<TimeSlotSelector />)).toMatchSnapshot();
+  test('should render correctly for multiple available time slots', () => {
+    expect(shallow(<TimeSlotSelector
+      party={2}
+      bookingsMadeToday={25}
+      availabilityInfo={[
+        { time: 17, remaining: 0 },
+        { time: 18, remaining: 5 },
+        { time: 19, remaining: 3 },
+        { time: 20, remaining: 2 },
+        { time: 21, remaining: 1 },
+      ]}
+      clickHandler={() => {}}
+    />)).toMatchSnapshot();
```

Comment:
I'm guessing you don't need to pass the actual function through here as it is only comparing the snapshot?

Reply:
yeah, TimeSlotSelector only passes its clickHandler down to each child TimeSlot component and never actually invokes it itself.  The TimeSlot component test makes sure the clickHandler is called properly.